docs/core/diagnostics/dotnet-trace.md

```diff
@@ -135,56 +139,66 @@ Lists pre-built tracing profiles with a description of what providers and filter
 dotnet-trace list-profiles [-h|--help]
 ```
 
-## Collect a trace with `dotnet-trace`
+## Collect a trace with dotnet-trace
 
-- To collect traces using `dotnet-trace`, you'll need to first, find out the process identifier (PID) of the .NET Core application to collect traces from.
+To collect traces using `dotnet-trace`:
 
-  - On Windows, there are options such as using the task manager or the `tasklist` command.
-  - On Linux, the trivial option could be using `ps` command.
+- Get the process identifier (PID) of the .NET Core application to collect traces from.
 
-You may also use the [dotnet-trace list-processes](#dotnet-trace-list-processes) command to find out what .NET Core processes are running, along with their PIDs.
+  - On Windows, for example, Task Manager or the `tasklist` command.
+  - On Linux, for example, the `ps` command.
+  - [dotnet-trace list-processes](#dotnet-trace-list-processes)
```

Comment:
This has been renamed to `dotnet-trace ps`.

Reply:
@Rick-Anderson this comment hasn't been addressed yet.